projects/angular/src/popover/common/_popover.clarity.scss

```
@@ -4,17 +4,24 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
+
 @use 'sass:map';
 @use '../../styles/mixins';
 @use '../../styles/variables/variables.global';
+@use '@angular/cdk/overlay-prebuilt.css';
 
 @include mixins.exports('popover.clarity') {
-  .clr-popover-content {
-    position: fixed;
-    z-index: map.get(variables.$clr-layers, smart-popover);
-  }
-
   .is-off-screen {
     @include mixins.off-screen-styles();
   }
+
+  //.clr-popover-content {
+  //  position: fixed;
+  //  z-index: map.get(variables.$clr-layers, smart-popover);
+  //}
+
+  .clr-container-element {
+    // position: absolute;
+    z-index: map.get(variables.$clr-layers, smart-popover);
+  }
```

Currently all the z-index properties on cdk are "1000" while ours have different layers system so we need to change these.
```suggestion
  .clr-popover-content {
    z-index: map.get(variables.$clr-layers, smart-popover);
  }

  .clr-overlay-container {
    &,
    .cdk-overlay-connected-position-bounding-box {
      z-index: map.get(variables.$clr-layers, smart-popover);
    }
  }
```

---

Corrected in commit ef34a7f